test_opensearchpy/test_async/test_server/test_plugins/test_alerting.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
```

Comment:
You can remove the modifications part from new files. That was added during the fork for files that had an ES license before.

Reply:
But the `utils/license-headers.py` enforce the `Modifications Copyright OpenSearch Contributors` part. Should I adapt checks made in the script ?

Reply:
Hmmm, we can deal with this separately.